diff --git a/FunctionBar.c b/FunctionBar.c
index 1c260ab..0a9a95c 100644
--- a/FunctionBar.c
+++ b/FunctionBar.c
@@ -17,6 +17,11 @@ in the source distribution for its full text.
 #include "ProvideCurses.h"
 #include "XUtils.h"
 
+static bool AltKeys = false;
+static const char* Alt_FunctionBar_FKeys[] = {" h", " S", " /", " \\", " t", " >", " ]", " [", " k", " q", NULL};
+void FunctionBar_useAltKeys(bool use_alt_keys) {
+   AltKeys = use_alt_keys;
+}
 
 static const char* const FunctionBar_FKeys[] = {"F1", "F2", "F3", "F4", "F5", "F6", "F7", "F8", "F9", "F10", NULL};
 
@@ -56,7 +61,7 @@ FunctionBar* FunctionBar_new(const char* const* functions, const char* const* ke
       this->size = i;
    } else {
       this->staticData = true;
-      this->keys.constKeys = FunctionBar_FKeys;
+      this->keys.constKeys = AltKeys ? Alt_FunctionBar_FKeys : FunctionBar_FKeys;
       this->events = FunctionBar_FEvents;
       this->size = ARRAYSIZE(FunctionBar_FEvents);
    }
diff --git a/FunctionBar.h b/FunctionBar.h
index 925e323..8c5f00a 100644
--- a/FunctionBar.h
+++ b/FunctionBar.h
@@ -20,6 +20,8 @@ typedef struct FunctionBar_ {
    bool staticData;
 } FunctionBar;
 
+void FunctionBar_useAltKeys(bool use_alt_keys);
+
 FunctionBar* FunctionBar_newEnterEsc(const char* enter, const char* esc);
 
 FunctionBar* FunctionBar_new(const char* const* functions, const char* const* keys, const int* events);
diff --git a/htop.c b/htop.c
index 00ff1da..6b93e6d 100644
--- a/htop.c
+++ b/htop.c
@@ -106,6 +106,7 @@ static CommandLineSettings parseArguments(int argc, char** argv) {
       {"no-unicode", no_argument,         0, 'U'},
       {"tree",       no_argument,         0, 't'},
       {"pid",        required_argument,   0, 'p'},
+      {"alt-keys",   no_argument,         0, 'a'},
       {"filter",     required_argument,   0, 'F'},
       {"highlight-changes", optional_argument, 0, 'H'},
       {0,0,0,0}
@@ -116,6 +117,9 @@ static CommandLineSettings parseArguments(int argc, char** argv) {
    while ((opt = getopt_long(argc, argv, "hVMCs:td:u::Up:F:H::", long_opts, &opti))) {
       if (opt == EOF) break;
       switch (opt) {
+         case 'a':
+            FunctionBar_useAltKeys(true);
+            break;
          case 'h':
             printHelpFlag();
             exit(0);
